docs/config/services/deployments/debug.js

```diff
@@ -10,30 +10,30 @@ module.exports = function debugDeployment(getVersion) {
       dependencyPath: '../../../'
     },
     scripts: [
-      '../angular.js',
-      '../angular-resource.js',
-      '../angular-route.js',
-      '../angular-cookies.js',
-      '../angular-sanitize.js',
-      '../angular-touch.js',
-      '../angular-animate.js',
-      'components/marked-' + getVersion('marked', 'node_modules', 'package.json') + '/lib/marked.js',
-      'js/angular-bootstrap/bootstrap.js',
-      'js/angular-bootstrap/dropdown-toggle.js',
-      'components/lunr.js-' + getVersion('lunr.js') + '/lunr.js',
-      'components/google-code-prettify-' + getVersion('google-code-prettify') + '/src/prettify.js',
-      'components/google-code-prettify-' + getVersion('google-code-prettify') + '/src/lang-css.js',
+      getComponentPath('hammerjs', 'hammer.js'),
+      getComponentPath('angular'),
+      getComponentPath('angular-animate'),
+      getComponentPath('angular-aria'),
+      getComponentPath('angular-material'),
+      getComponentPath('angular-sanitize'),
+      getComponentPath('marked', 'lib/marked.js', 'node_modules', 'package.json'),
+      getComponentPath('lunr.js', 'lunr.js'),
+      getComponentPath('google-code-prettify', 'src/prettify.js'),
+      getComponentPath('google-code-prettify', 'src/lang-css.js'),
       'js/versions-data.js',
       'js/pages-data.js',
       'js/nav-data.js',
       'js/docs.js'
     ],
     stylesheets: [
-      'components/bootstrap-' + getVersion('bootstrap') + '/css/bootstrap.css',
-      'components/open-sans-fontface-' + getVersion('open-sans-fontface') + '/open-sans.css',
+      getComponentPath('angular-material', 'angular-material.css'),
+      getComponentPath('angular-material', 'themes/grey-theme.css'),
+      getComponentPath('angular-material', 'themes/red-theme.css'),
       'css/prettify-theme.css',
       'css/docs.css',
-      'css/animations.css'
+      'css/animations.css',
+      'font-awesome/css/font-awesome.css',
+      '//fonts.googleapis.com/css?family=Roboto:400,400italic,500,700'
```

Comment:
I think there was some discussion in `ngMaterial` about whether they should automatically import the `Roboto` font (but I am not sure what is the current status). This might or might not be needed.